src/installer/pkg/sfx/installers/dotnet-runtime-deps/dotnet-runtime-deps-cm.2.proj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <GenerateInstallers Condition="'$(BuildRpmPackage)' != 'true'">false</GenerateInstallers>
+    <PackageTargetOS>cm.2</PackageTargetOS>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <LinuxPackageDependency Include="openssl-libs;icu;krb5" />
```

Comment:
Q: do we need libnuma? It is a soft (!required) dependency, but some functionality in coreclr takes advantage of it for scaling / multi-processor scenarios (@janvorli, please correct me if that's not the case).

Reply:
> Q: do we need libnuma? It is a soft (!required) dependency, but some functionality in coreclr takes advantage of it for scaling / multi-processor scenarios (@janvorli, please correct me if that's not the case).

We can update this later. For now, we'd like to use the same set of dependencies as Mariner 1.0.